src/App.js

```diff
@@ -0,0 +1,25 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import BookStore from './components/BookStore';
+import Header from './components/Header';
+import Categories from './components/Categories';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <>
+              <Header />
+              <BookStore />
+            </>
+)}
+        />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </BrowserRouter>
+  );
```

Comment:
- [ ] [OPTIONAL] I assume that the Header component should be static and not dependent on Routes, so it should be something like.

```
function App() {
  return (
    <BrowserRouter>
      <Header />
      <Routes>
        <Route
          path="/"
          element={(
            <>
              <BookStore />
            </>
)}
        />
        <Route path="/categories" element={<Categories />} />
      </Routes>
    </BrowserRouter>
  );
}
```
Because the current case makes the Header disappear when the Categories link is clicked on, but it should be there to enable clicking back to the Books component section.